proto/rill/runtime/v1/resources.proto

```diff
@@ -425,18 +426,17 @@ message DashboardSpec {
   string title = 1;
   uint32 columns = 2;
   uint32 gap = 3;
-  repeated DashboardComponent components = 4;
+  repeated DashboardItem items = 4;
 }
 
 message DashboardState {}
 
-message DashboardComponent {
-  string chart = 1;
+message DashboardItem {
+  string component = 1;
```

Comment:
After fetching the dashboard, each item's component must be fetched separately using `GetResource`. Is that okay, or would you prefer if we added an API for fetching a dashboard and all of its components in a single request? It could look something like this:
```proto
message GetDashboardRequest {
  string instance_id = 1;
  string name = 2;
}

message GetDashboardResponse {
  Resource dashboard = 1;
  map<string, Resource> components = 2;
}
```

Reply:
It would only be advantageous if a new resource is not emitted for each inline resource. But all inlined resources would get invalidated if one changes in that case, so i think this is preferable.

Reply:
I was thinking that the response `components` would return *every* referenced component (both inline and non-inline). So they could still be treated the same. Does that change your assessment or not?

Reply:
It would probably help in caching for non-inline components. So not sending it inline is better.

Reply:
Yeah, I don't mind this. It might simplify the read only versions of the dashboards, but it's fine as is.

Reply:
Got it, won't add this then :)